wasmedge-ggml-llama-interactive/README.md

```diff
@@ -1,10 +1,42 @@
 # Llama Example For WASI-NN with GGML Backend
 
-## Dependencies
+## Requirement
 
-Install the latest wasmedge with plugins (not released yet)
+### For macOS (apple silicon)
 
-## Build
+1. Install WasmEdge 0.13.4 via installer
+
+```bash
+curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash
+# After install the wasmedge, you have to activate the environment.
+# Assuming you use zsh (the default shell on macOS), you will need to run the following command
+source $HOME/.zshenv
+```
+
+2. Install plugins bundled in this repo
```

Comment:
Instead of putting the plugins here locally, can we go to the 0.13.4 release and manually add those plugins in it? Would it make the --plugin wasi-nn-ggml install command working?

If we still want to put the plugin in this repo, we should put all the ggml plugins here. I would suggest creating a temp release in this repo and then upload all those plugins in that case.

Reply:
I just uploaded the manually pre-built wasi_nn-ggml plugin for the macOS Apple silicon model.
Need to merge https://github.com/WasmEdge/WasmEdge/pull/2872 first.

Reply:
![CleanShot 2023-09-22 at 21 15 04](https://github.com/second-state/WasmEdge-WASINN-examples/assets/2776756/e1d39f04-811a-4bb1-b183-ec5feca6eb1d)
Done.